README.md

```diff
@@ -302,9 +308,35 @@ The easiest option is using OAuth with your Google Account. Summarized instructi
 ```
 gcloud auth application-default login --scopes=https://www.googleapis.com/auth/bigquery,https://www.googleapis.com/auth/iam.test
 ```
+
+The `profiles.yml` file included in this package should be removed. The `profile: 'default'` line in `dbt_project.yml` in this package should also be removed.
```

Comment:
@davidbooke4 I understand that these files aren't needed by a package user, but what is the benefit of removing them? They'll be added back the next time the person runs dbt-deps, right?

Reply:
Yep, you're right - I forgot that would be the case! I'll update the comments I made in the README based on that fact.

One consequence of defining some of these project variables in `dbt_project.yml` within the package is that these variables will carry through to someone's dbt project if they've installed dbt-ga4 but haven't set new values for these variables in their own `dbt_project.yml`. So models such as `stg_ga4__page_conversions` and `stg_ga4__derived_session_properties` will be enabled and have fields created based on the variables I've added to `dbt_project.yml` in the package.

I'm trying to do some exploring to see if there's an alternative, but what are your thoughts on that? Do you think that'd be okay or do we need to find a way so these models aren't enabled if someone doesn't set those variables?

Reply:
Shoot, yea that's a problem because I don't think many package users use those advanced features or set those variables. 

Can you look at alternatives?

Reply:
I found an alternative that boils down to creating more repo environment variables and updating the conditional enabled/disabled logic in some models to look for those environment variables.

However, now I'm back to an earlier problem where the project is unable to compile because unit tests are defined for models that are disabled. There's an open [issue](https://github.com/dbt-labs/dbt-core/issues/9109) and [PR](https://github.com/dbt-labs/dbt-core/pull/10831) related to this (with commits as recent as last week), so a fix should be in place soon 🤞. 

What are your thoughts on waiting for this fix to be in place before merging this PR? The alternative would be to remove the dbt unit tests and add the `pytest` tests back in for the few models that are enabled based on setting the  conversion events and derived user/session properties variables. I'd add in a TODO for replacing those `pytest` tests with unit tests if we decide to proceed with that.

Reply:
Hey @adamribaudo-velir! I wanted to give the update that the PR I mentioned in my comment above was merged last week. That means we're probably at least a couple weeks out before it's included in a dbt version release, but I don't think there's any rush to get my PR merged so that should be fine. Let me know what you think though!

Reply:
(Initially replied in the main thread)

Sounds great! Yes, let's wait until this is released in dbt-core. Thanks for staying on top of it.